src/core/exec-invoke.c

```diff
@@ -2270,20 +2270,55 @@ static int setup_private_users_child(int unshare_ready_fd, const char *uid_map,
         return 0;
 }
 
+static int bpffs_helper(const ExecContext *c, int socket_fd) {
+        char number[STRLEN("0x") + sizeof(c->bpf_delegate_commands) * 2 + 1];
+
+        assert(c);
```

Comment:
assert needs to go before the definition above

Reply:
Yeah, looking at the assembly, without optimization, gcc just creates space on the stack. So I think this is fine.

Reply:
Interesting, but will move the assertion at the beginning of the function.